### ISSUE TITLE
Cannot reproduce experimental results on Openlane?

### ISSUE THREAD

**Issue by inconnu11** [gigo-team/bev_lane_det#10]
Hello all,

I ran the Openlane experiment based on the provided codebase and didn't change any parameters in the config file. I tested multiple models saved at different epochs and only got the best results of 56.3% F-score on Openlane validation set, which is still 2% lower than the number shown in your paper (i.e., 58.4% F-score).  Is there anything wrong? Could you please give me some advice?

**Comment by qinjian623** [comment]
hi,

You can try adjusting the training hyperparameters, as different numbers of GPUs and hardware environments may yield different results. 

It is recommended to adjust the learning rate (lr) and batch size.

Using this version of the open-source code, we can achieve F1 score of 60.

**Comment by yuaoze** [comment]
> hi,
> 
> You can try adjusting the training hyperparameters, as different numbers of GPUs and hardware environments may yield different results.
> 
> It is recommended to adjust the learning rate (lr) and batch size.
> 
> Using this version of the open-source code, we can achieve F1 score of 60.

hi,
I've tried and get the best result of F-score is 57.4% on Openlane, My device is 2x3090, batch_size is 32, other hyperparameters are same as the code. It shows that loss oscillated around 5 without decline. Could you please give me some advice?

**Comment by secret104278** [comment]
HI, @inconnu11 and @yuaoze, is there any possible that you could share your own trained model for quick evaluation?

**Comment by qinjian623** [comment]
hi, @yuaoze 

We use a hardware configuration of 8 x V100, which you can use as a reference to adjust hyperparameters.

**Comment by yuaoze** [comment]
> 

sure, please leave your email

**Comment by Laulian** [comment]
> > 
> 
> sure, please leave your email
@yuaoze 
lanlian919@gmail.com ~ It will be greatly appreciated

**Comment by secret104278** [comment]
> sure, please leave your email

secret104278@gmail.com

thanks a lot ~

**Comment by zhanghui75** [comment]
Could you also kindly share your trained model with me? Thanks a lot.
My email: zhh9297@163.com

**Comment by imyyf** [comment]
> > hi,
> > You can try adjusting the training hyperparameters, as different numbers of GPUs and hardware environments may yield different results.
> > It is recommended to adjust the learning rate (lr) and batch size.
> > Using this version of the open-source code, we can achieve F1 score of 60.
> 
> hi, I've tried and get the best result of F-score is 57.4% on Openlane, My device is 2x3090, batch_size is 32, other hyperparameters are same as the code. It shows that loss oscillated around 5 without decline. Could you please give me some advice?

Hi
which epoch you choose to get 57.4% on openlane? we use epoch049(batch_size is 32) to test, only get 53%.

**Comment by adeelahmedrana** [comment]
Hi, first of a very impressive paper.
Can you please share the weights with me too at adeel0311@gmail.com

Many thanks

**Comment by Lyric0620** [comment]
@yuaoze 
Could you please share your weights with me just for a early demo. Thanks a lot.

My email: 3118891520@qq.com
                yutongliu840@gmail.com

**Comment by westlife35** [comment]
@yuaoze 
Could you please share your weights(57.4% on openlane). Thanks~
My email: 2364426805@qq.com

**Comment by superchenyan** [comment]
> > 
> 
> sure, please leave your email

my email:  superchenyan@sina.com  ths!!!

**Comment by drilistbox** [comment]
> 

could you share your weight to me, my email is drilistbox@buaa.edu.cn

**Comment by PyNancy** [comment]
> > 
> 
> sure, please leave your email

could you please share your weight to me, thanks a lot, guoweiyang8199@gmail.com

**Comment by WwwwYz666** [comment]
Could you please share your checkpoint weight ? My email is [wangyic19@gmail.com](wangyic19@gmail.com).

**Comment by CaiFuyao** [comment]
> 

could you please share your best model to me , thanks a lot! My email is cfycfy@hust.edu.cn.

**Comment by s95huang** [comment]
Here is what I got,
{'f1_score': 0.5651117116621163,
 'precision': 0.6838964779914308,
 'recall': 0.4814843714408606,
 'x_error_close': 0.25393218394610845,
 'x_error_far': 0.6785732514552418,
 'z_error_close': 0.19615825120228755,
 'z_error_far': 0.6227202588181409}

**Comment by Pocan0** [comment]
> Here is what I got, {'f1_score': 0.5651117116621163, 'precision': 0.6838964779914308, 'recall': 0.4814843714408606, 'x_error_close': 0.25393218394610845, 'x_error_far': 0.6785732514552418, 'z_error_close': 0.19615825120228755, 'z_error_far': 0.6227202588181409}

could you please share your  model to me , thanks a lot! My email is cshenwhut@126.com.

**Comment by Mollylulu** [comment]
@yuaoze Could you please share your reproduced model weights with me? Thanks a lot! 
Email address: `gracegreen77@163.com`.

**Comment by EnternalTwinkle** [comment]
@yuaoze Can you share the source code? Thank you very much，1017094591@qq.com

**Comment by Ywenjun123** [comment]
> Hello all,
> 
> I ran the Openlane experiment based on the provided codebase and didn't change any parameters in the config file. I tested multiple models saved at different epochs and only got the best results of 56.3% F-score on Openlane validation set, which is still 2% lower than the number shown in your paper (i.e., 58.4% F-score). Is there anything wrong? Could you please give me some advice?

 could you please share your model to me , thanks a lot! My email is 1119782358@qq.com

**Comment by happyday-lkj** [comment]
> Here is what I got, {'f1_score': 0.5651117116621163, 'precision': 0.6838964779914308, 'recall': 0.4814843714408606, 'x_error_close': 0.25393218394610845, 'x_error_far': 0.6785732514552418, 'z_error_close': 0.19615825120228755, 'z_error_far': 0.6227202588181409}

could you share your pretrained model, happyday_lkj@163.com

**Comment by spring35** [comment]
> > 
> 
> sure, please leave your email

could you share your model, thanks! my email is 2893102339@qq.com

**Comment by Benson722** [comment]
Could you also kindly share your trained model with me? Thanks a lot.
My email: 583032099@qq.com
thx!

**Comment by dada-h-h** [comment]
@yuaoze Hi, could you kindly share your model weights with me? My email is dhkim@mcl.korea.ac.kr. I'd really appreciate it. Thanks so much!

**Comment by vietanhdsai** [comment]
@yuaoze Could you please share your model weights with me? My email is vietanh040902@gmail.com. Thank you very much.
